pmd-apex/src/main/resources/category/apex/design.xml

```diff
@@ -408,6 +408,37 @@ public class Person {
     Date birthDate;
     BodyMeasurements measurements;
 }
+]]>
+        </example>
+    </rule>
+
+    <rule name="AvoidUnusedMethod"
```

Comment:
Let's keep the alphabetical order of the rules in the file and move this rule declaration up.